database-creation.py

```diff
@@ -0,0 +1,32 @@
+import mariadb
+import sys
+#run this command in linux to start the docker container with maria db
+    #docker run -d -p 3306:3306 -e MYSQL_DATABASE=example -e MYSQL_ROOT_PASSWORD=password -e TZ=America/Los_Angeles --name mdb103 mariadb:10.3
+
+global curr
```

Comment:
global variables should be declared at the module level and made internal to the module by prepending an _ to the name as per the google python style guide

Reply:
will be done in a later iteration this is just for the prototype